src/goto-instrument/contracts/contracts.cpp

```diff
@@ -734,6 +733,29 @@ code_contractst::create_ensures_instruction(
   return std::make_pair(std::move(ensures_program), std::move(history));
 }
 
+static const code_with_contract_typet &
+get_contract(const irep_idt &function, const namespacet &ns)
+{
+  const std::string function_str = id2string(function);
+  const symbolt &function_symbol = ns.lookup(function);
```

Comment:
[nit] could use `auto` here
```suggestion
  const auto function_str = id2string(function);
  const auto &function_symbol = ns.lookup(function);
```